.github/workflows/maven.yml

```diff
@@ -103,6 +103,8 @@ jobs:
           semantic-release --prepare @conveyal/maven-semantic-release --publish @semantic-release/github,@conveyal/maven-semantic-release --verify-conditions @semantic-release/github,@conveyal/maven-semantic-release --verify-release @conveyal/maven-semantic-release --use-conveyal-workflow --dev-branch=dev --skip-maven-deploy
       # The git commands get the commit hash of the HEAD commit and the commit just before HEAD.
       - name: Prepare deploy artifacts
+        # Only deploy on push (pull_request will deploy a temp. merge commit. See #400.)
+        if: github.event_name == 'push'
         run: |
           # get branch name of current branch for use in jar name
           export BRANCH=$GITHUB_HEAD_REF_SLUG
```

Comment:
Interesting, I don't see a jar with the branch name in the datatools-builds S3 bucket. Are we using the wrong git variable, and maybe it should be `GITHUB_REF_SLUG`? (see https://github.com/rlespinasse/github-slug-action#slug-variables)

Reply:
Yeah seems like `GITHUB_REF_SLUG` will make more sense. I'll give it a shot and we'll see if it arrives correctly. Thanks for noticing that and letting me know!